example/dj/apps/app/tests/data_processors.py

```diff
@@ -68,17 +67,15 @@ def test_error_during_creating_user_with_file_and_not_defined_content_type(self,
     def serve_file(self, rsps, url, body, status=200):
         rsps.add(responses.GET, url, body=body, status=status, match_querystring=True)
 
-    def get_file_url_response(self, data):
-        url = 'http://foo.bar/testfile.pdf'
+    def get_file_url_response(self, data, call_url=None, serve_url='http://foo.bar/testfile.pdf'):
         data['contract'] = {
             'filename': 'testfile.pdf',
-            'url': url,
+            'url': call_url or serve_url,
         }
-        with responses.RequestsMock(assert_all_requests_are_fired=True) as rsps:
+        with responses.RequestsMock(assert_all_requests_are_fired=(call_url == serve_url)) as rsps:
```

Comment:
tohle není podle mě správné řešení

Reply:
měl bys připravovat 404 pokud to má 404 vrátit nebo responses vůbec nenasatovat pokud je url nevalidní